superset/assets/src/dashboard/components/filterscope/FilterScopeModal.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ModalTrigger from '../../../components/ModalTrigger';
+import FilterScope from '../../containers/FilterScope';
+
+const propTypes = {
+  triggerNode: PropTypes.node.isRequired,
+};
+
+export default class FilterScopeModal extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.modal = null;
+    this.close = this.close.bind(this);
+    this.setModalRef = this.setModalRef.bind(this);
+  }
+
+  setModalRef(ref) {
+    this.modal = ref;
+  }
+
+  close() {
+    this.modal.close();
```

Comment:
can you add a check that `this.modal` is defined?

Reply:
fixed.